app/templates/select.hbs

```diff
@@ -10,23 +10,14 @@
     {{/if}}
   </a>
 </div>
-
-<div {{bind-attr class=":dropdown-menu :js-dropdown-menu dropdownMenuClass
-  isDropdownMenuPulledRight:pull-right"}}>
+<div {{bind-attr class=":dropdown-menu :js-dropdown-menu dropdownMenuClass isDropdownMenuPulledRight:pull-right"}}>
```

Comment:
The dropdownMenuClass property was not being read/substituted because of the new line